salt/modules/elasticsearch.py

```diff
@@ -174,7 +174,7 @@ def alias_get(indices=None, aliases=None, hosts=None, profile=None):
     return None
 
 
-def document_create(index, doc_type, body=None, hosts=None, profile=None):
+def document_create(index, doc_type, body=None, id=None, hosts=None, profile=None):
```

Comment:
@rallytime: yeah, I thought about that. The API changed completely in 2015.8.0, so there should not be any code using it yet (I know "should" is a big word). I put `id` there for consistency with the other routines in this module. If you think it should be moved to the end, I will change it.